vantage/README.md

```diff
@@ -0,0 +1,27 @@
+# Vantage
+
+## Overview
+
+Vantage is a cloud cost transparency and optimization platform. This integration allows Datadog users to import their Datadog costs into Vantage and track alongside their other infrastructure spend, such as AWS, Snowflake, and Kubernetes.
+
+### Uninstallation
```

Comment:
What do you think about moving the `Uninstallation` section closer to the bottom? After `Configuration` and before `Support`. I think the doc will flow better with the `Installation` section up top.

Reply:
I'm good with this suggestion - having this show up under the Configure tab makes more sense

Reply:
From a documentation perspective, I agree that it makes more sense to relocate the `Uninstallation` section as mentioned here.